app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -164,7 +162,7 @@ class SuggestedEditsTasksFragment : Fragment() {
         multilingualTeaserTask.description = getString(R.string.suggested_edits_task_multilingual_description)
         multilingualTeaserTask.showImagePlaceholder = false
         multilingualTeaserTask.showActionLayout = true
-        multilingualTeaserTask.disabled = !Prefs.isSuggestedEditsTranslateDescriptionsUnlocked()
+        multilingualTeaserTask.disabled = false
```

Comment:
You can just simply remove it. The default value of `SuggestedEditsTask.disabled` is `false`.